suites/apis/centralizedAuth.js

```diff
@@ -154,19 +154,48 @@ const indexed_files = {
     md5: '73d643ec3f4beb9020eef0beed440ae9',
     authz: [
       '/gen3/programs/test_program/projects/test_project1',
-      '/gen3/consents/GRU'
+      '/consents/GRU'
     ],
     size: 42,
   },
-  hmbResearchFile: {
+  gen3HmbResearchFile: {
     filename: 'testdata',
     link: 's3://cdis-presigned-url-test/testdata',
     md5: '73d643ec3f4beb9020eef0beed440af1',
     authz: [
       '/gen3/programs/test_program/projects/test_project1',
-      '/gen3/consents/HMB'
+      '/consents/HMB'
     ],
     size: 43,
+  },
+  abcHmbResearchFile: {
+    filename: 'testdata',
+    link: 's3://cdis-presigned-url-test/testdata',
+    md5: '73d643ec3f4beb90213ef0beed440af1',
+    authz: [
+      '/abc/programs/test_program/projects/test_project1',
+      '/consents/HMB'
+    ],
+    size: 44,
+  },
+  gruResearchFile: {
```

Comment:
i think this file is the same as `generalResearchUseFile`? and `generalResearchUseFile` is unused. could you remove one of the 2?

Reply:
oh, yeah, whoops :man_facepalming: